### DISCUSSION TITLE
How to initiate Aurora v1 storage?

### DISCUSSION THREAD

**Discussion by rydes01** [aws/copilot-cli#4943] [Q&A] [answered]
We are trying to use Aurora v1 for a project's dev database to take advantage of the pause/resume feature not found in Aurora v2 to save hopefully some money. But we are having trouble activating the v1 storage and instead keep getting Writer Instances that say "Serverless v2 (0.5 - 8 ACUs)". This is Aurora v2 as I understand it.

The steps we have taken are:
  - copilot app init --domain example.com
  - copilot init 
  - copilot env init
  - copilot env deploy --name dev
  - copilot storage init -n NAME -t Aurora --serverless-version v1
  - copilot env deploy

This is taken from the [Storage page](https://aws.github.io/copilot-cli/docs/developing/storage/) of the documentation.

Any help would be appreciated.

**Comment by rydes01** [comment]
We're using v1.22 of copilot since that was the last version before Aurora v2 was introduced. This is not ideal, but is working and has created an Aurora v1 cluster that spins down as expected.

I wonder if the cli is not recognising the 'v1' argument and silently using the default of v2? I tried using uppercase V but that does cause an error.

**Reply by dannyrandall** [reply, marked as answer]
Hey @ryandeslandes! I just looked into this, and it looks like we don't actually have a template to create an _environment_ level Aurora v1 addon - only _service_ level. I'm creating an issue so we can fix this! (https://github.com/aws/copilot-cli/issues/4960)

**Reply by rydes01** [reply]
Thanks @dannyrandall! I will mark this as answered and we'll await the bugfix.
